tests/cxx/no_fwd_decls.cc

```diff
@@ -41,6 +43,13 @@ bool AreSame(const NameOnly& a, const NameOnly& b) {
   return AddressOf(a) == AddressOf(b);
 }
 
+// ClassInSrc is forwad declared in no_fwd_decls-fwd2.h, so IWYU must keep that
+// include even though the definition can be found in this source file, but
+// after the use.
+void ForwardDeclareUse(const ClassInSrc&);
+
+class ClassInSrc {};
```

Comment:
Actually, instead of introducing a new class and function, couldn't you just define `Fwd` below the first `ForwardDeclareUse`, and add a comment to explain why?